CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Scala: parse typed patterns with variables that begin with an underscore: 
   `case _x : Int => ...`
 - Scala: parse unicode identifiers
+- semgrep-core accepts `sh` as an alias for bash
```

Comment:
The changes to the changelog look suspicious (there are too many and unrelated), and this particular line and the next one are duplicated.

Reply:
weird. I did `git fetch; git merge origin/develop` and didn't check the result.

Reply:
I know @nbrahms added a git setting to help solve the merge conflicts on the changelog file. I guess here what happened is that I edited one of the existing lines, and instead of replacing the old one during the merge, it was added with the rest of my additions.